steps/setup_docker_skills_selection.yml

```diff
@@ -27,6 +27,8 @@ startFlow:
                     label: 'Communication'
                   - value: 'Documentation'
                     label: 'Documentation'
+                  - value: 'git'
```

Comment:
upper case? All the other options are upper cased

Reply:
I was actually wondering about it, since in the database the value is lowercase

Reply:
@NivRichter ?

Reply:
@esapir both work, I changed it 👍